### ISSUE TITLE
Fix file size calculations in Vivaldi

### ISSUE THREAD

**Issue by Aldaviva** [Aldaviva/VivaldiCustomLauncher#15]
Particularly in downloads, the data sizes calculated by Vivaldi are incorrect. They mistakenly convert 1000 B = 1KB and 1000 KB = 1 MB, when the correct conversion is 1024 B = 1 KB and 1024 KB = 1 MB. Example:
![image](https://github.com/user-attachments/assets/c0f648d4-d135-477c-9fdb-5b04ecc26797)

This file is actually 695,472,128 bytes, which Windows correctly reports as 663 MB. However, Vivaldi shows it as 695.47 MB, because it's 695 million bytes.

There are probably lots of additional places that Vivaldi shows normalized data sizes, such as download prompts and email attachments, but I am most interested in fixing the downloads pane if possible.

**Comment by Aldaviva** [comment]
This seems more like a bug than a misunderstanding or disagreement in Vivaldi. Here is snippet that formats data sizes, formatted:
```js
var gh = n(86461);
const mh = [(0, nt.Z)("B"), (0, nt.Z)("kB"), (0, nt.Z)("MB"), (0, nt.Z)("GB"), (0, nt.Z)("TB"), (0, nt.Z)("PB"), (0, nt.Z)("EB"), (0, nt.Z)("ZB"), (0, nt.Z)("YB")], vh = (0, nt.Z)("bytes");
function fh(e, t, n = !0, i = !1)
{
    if (isNaN(e))
        return e ? e.toString() : "";
    if (e <= 0)
        return "0 " + (n ? vh : mh[0]);
    const s = i ? 1024 : 1e3,
    o = t ?? (e < s ? 0 : 2);
    let a = mh.length - 1;
    for (var r = 0; r < mh.length; r++)
        if (e < Math.pow(s, r + 1))
        {
            a = r;
            break
        }
    let l = (e / Math.pow(s, a)).toLocaleString(void 0,
    {
        maximumFractionDigits: o
    }
    );
    if (o > 0)
    {
        const e = "." + "0".repeat(o);
        l.endsWith(e) && (l = l.substring(0, l.length - e.length))
    }
    const d = 0 === a && n ? vh : mh[a];
    return l + (d ? " " + d : "")
}
var yh = n(29526);
```

It uses base-1024 if the argument `i` is `true`, or base-1000 if `i` is `false` (the default). For some reason, `i` is being passed as `false` or being omitted, so it uses the wrong base. Maybe this is a new argument and some callers weren't updated?

Given this, options to proceed are
1. wait for Vivaldi to fix this upstream and ignore the issue until then
2. make a nuisance of myself and report this in the Vivaldi forums
3. patch `bundle.js` to change the behavior of this `fh` function, possibly negatively affecting other callers
4. patch multiple callers to pass the correct `i` argument

I think I might do 1 until the next major release (there is [one with a large reskin coming](https://vivaldi.com/blog/desktop/desktop-snapshots/new-look-new-feature-vivaldi-browser-snapshot-3483-4/)), and if it isn't fixed by then, do 3.

**Comment by Aldaviva** [comment]
[This was apparently changed from 1024 to 1000 in version 6.2 and seems to be a feature, not a bug.](https://forum.vivaldi.net/topic/92075/option-to-select-binary-or-decimal-units-for-file-sizes/5) I'm still not sure why it's an argument in the function if there is no way to change the value of that argument, but that's fine. Maybe 3 is the best course of action overall.

**Comment by Aldaviva** [comment]
Success. Changing the default value of the `i` argument worked.
![image](https://github.com/user-attachments/assets/4fbedfb2-cea0-465e-b2ae-e6da77f7ca68)

**Comment by Aldaviva** [comment]
Released in 1.3.9.
